packages/store-api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -28,18 +37,56 @@ export const VtexCommerce = (opts: Options) => {
     checkout: {
       simulation: (
         args: SimulationArgs,
-        options: SimulationOptions = { sc: '1' }
+        simulationOptions: SimulationOptions = { sc: channel }
```

Comment:
`sc` is `salesChannel`? if yes, can we use the full name instead of the initials?

Reply:
I agree with